moon/cfc_77.15.580/server/connect.moon

```diff
@@ -0,0 +1,24 @@
+gameevent.Listen "player_connect"
+
+hook.Add "player_connect", "Section580_ConnectionThrottle", (data) ->
+    { :networkid, :userid, :name } = data
+
+    connections = Section580.connectSpam[networkid]
+
+    if not connections
+        Section580.connectSpam[networkid] = 1
+        return
+
+    Section580.connectSpam[networkid] += 1
+
+    if Section580.connectSpam[networkid] > Section580.connectSpamThreshold
+        reason = "Too many connnections"
+
+        game.KickID userid, reason
+
+        return unless Section580.connectShouldBan
+
+        -- FIXME: Expect ULib at this point
```

Comment:
is this not just 
`return unless ULib`
?

Reply:
No, I definitely don't want this to silently do nothing. Either ULib exists and they get banned, or it doesn't and it errors